core/appeal/service.go

```diff
@@ -1717,9 +1716,10 @@ func getProvider(a *domain.Appeal, providersMap map[string]map[string]*domain.Pr
 }
 
 func validateAppeal(a *domain.Appeal, pendingAppealsMap map[string]map[string]map[string]*domain.Appeal) error {
-	if pendingAppealsMap[a.AccountID] != nil &&
-		pendingAppealsMap[a.AccountID][a.ResourceID] != nil &&
-		pendingAppealsMap[a.AccountID][a.ResourceID][a.Role] != nil {
+	accountID := strings.ToLower(a.AccountID)
+	if pendingAppealsMap[accountID] != nil &&
```

Comment:
we need to make sure that `pendingAppealsMap` values are assigned using lowercase email as well

Reply:
Yes, this is already handled on `getAppealsMap` function